presto-hive/src/test/java/io/prestosql/plugin/hive/metastore/cache/TestCachingHiveMetastore.java

```diff
@@ -263,6 +270,25 @@ public void testNoCacheExceptions()
         assertEquals(mockClient.getAccessCount(), 2);
     }
 
+    @Test
+    public void testCachingHiveMetastoreCreationWithTtlOnly()
+    {
+        CachingHiveMetastoreConfig config = new CachingHiveMetastoreConfig();
+        config.setMetastoreCacheTtl(new Duration(10, TimeUnit.MILLISECONDS));
+
+        CachingHiveMetastore metastore = createMetastore(config);
+
+        assertThat(metastore).isNotNull();
+    }
+
+    private CachingHiveMetastore createMetastore(CachingHiveMetastoreConfig config)
```

Comment:
I would inline this method. It creates metastore with direct executor and so it makes it usable only from one context (the test above).

Alternatively, please rename createMetastore -> createMetastoreWithDirectExecutor

Reply:
I've renamed the method as it sounds ok for the test purpose